src/main/scala/com/gu/rcspollerlambda/services/Kinesis.scala

```diff
@@ -16,11 +18,23 @@ object Kinesis extends Logging {
     rcsUpdates.map(publish).sequence_
   }
 
+  private val compressionMarkerByte: Byte = 0x00.toByte
```

Comment:
Shouldn't this be `0x01` for GZIP in alignment with other places we do this?
https://github.com/guardian/content-api/blob/086287cd231e4072b096a6815193f8478f49088f/porter/src/main/scala/com.gu.contentapi.porter/integration/ThriftDeserializer.scala#L35-L44

Reply:
Oh, I wasn't aware we did this. I based this on [Grid](https://github.com/guardian/grid/blob/master/common-lib/src/main/scala/com/gu/mediaservice/lib/json/JsonByteArrayUtil.scala#L14).